core/unix/os.c

```diff
@@ -4702,7 +4699,7 @@ void
 os_syslog(syslog_event_type_t priority, uint message_id, uint substitutions_num,
           va_list args)
 {
-    int native_priority;
+    int native_priority UNUSED;
```

Comment:
What does the native_priority var here achieve?

Reply:
Didn't want to deal with the unfinished code: you're asking to delete the function body code that just sets native_priority and never does anything with it?

Reply:
Going to leave it.